packages/fxa-admin-panel/src/components/Nav/index.tsx

```diff
@@ -12,13 +12,33 @@ export const Nav = () => (
       <h2>Navigation</h2>
       <ul>
         <li>
-          <NavLink exact to="/email-blocks">
+          <NavLink exact to="/account-search">
             <img
               className="inline-flex icon"
               src={require('../../images/icon-mail.svg')}
               alt="external link"
             />
-            Email blocks
+            Account Search
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to="/site-status">
+            <img
+              className="inline-flex icon"
+              src={require('../../images/icon-site-status.svg')}
```

Comment:
I realize you're following the existing pattern from the mail icon here but I think we should prefer `import` over `require` when we can, so I'd like to change this to something like this:

```jsx
import './index.scss';
import mailIcon from '../../images/icon-mail.svg';
import statusIcon from '../../images/icon-site-status.svg';

...

<img src={statusIcon} ... />
```